### PR TITLE
Create team draw emails

### LINKED ISSUE #792: Email draw notifications: Send email to teams on round release
Continued from #671. The teams should get a notification with the venue (& round), their position (if applicable), opponent(s), and adjudicators for the match.

### PR DESCRIPTION
**Pull request by tienne-B**
Check commit messages.

Closes #792.

### CODE REVIEW

**Comment by philipbelesky** [comment]
Looks good! All merged — feel free to patch the CHANGELOG as needed btw.

**Comment by philipbelesky** [comment]
@tienne-B note the fixes in dbb0009d9ac0a5f60ff66a5bcf052cbb7e2ec084. I should have paid more attention to the CI results, although it might also be worth setting up a git hook for `flake8` on your end also?

**Comment by tienne-B** [comment]
Thanks!

File: tabbycat/draw/urls_admin.py [hunk 1]
```diff
@@ -29,9 +29,12 @@
             name='draw-regenerate'),
 
         # Email
-        path('email/',
+        path('email-adjudicators',
             views.EmailAdjudicatorAssignmentsView.as_view(),
-            name='draw-email'),
+            name='draw-adj-email'),
+        path('email-debaters',
+            views.EmailAdjudicatorAssignmentsView.as_view(),
```
Comment (czlee) [inline tabbycat/draw/urls_admin.py]: Was this meant to be `EmailTeamAssignmentsView`?

Comment (tienne-B) [reply tabbycat/draw/urls_admin.py]: #995.

**Pull request merged**